config/application.rb

```diff
@@ -67,14 +67,16 @@ class Application < Rails::Application
     config.contentful_management_access_token = ENV.fetch('CONTENTFUL_MANAGEMENT_TOKEN', credentials.dig(:contentful, :management_access_token)) # TODO: use service account management token
     config.contentful_environment             = ENV.fetch('CONTENTFUL_ENVIRONMENT', credentials.dig(:contentful, :environment))
 
+    config.active_record.schema_format = :sql
+
     # @return [Boolean]
     def live?
       ENV['WORKSPACE'].eql?('production')
     end
 
     # @return [Boolean] true if Contentful is used for training content
     def cms?
-      Types::Params::Bool[ENV.fetch('CONTENTFUL', true)]
+      Types::Params::Bool[ENV.fetch('CONTENTFUL', false)]
```

Comment:
Instead I was proposing we make the change in the CI workflow envs which I've done in one of my PRs.

Reply:
It'll make more noise in the code that way. One line in the `app_config.yml` for prod to disable it and same again in CI.